node/src/components/consensus.rs

```diff
@@ -71,6 +76,11 @@ pub enum Event<I> {
         sender: I,
         proto_block: ProtoBlock,
     },
+    /// Response from the Contract Runtime, containing the validators for the new era
+    GetValidatorsResponse {
+        switch_block_header: BlockHeader,
```

Comment:
Is it necessary to include `switch_*` in the name?

Reply:
Probably not necessary, but I think it expresses the purpose of the field more clearly. I'm not attached to it, though.